File: Demo/app/src/main/java/com/example/mhamed/demo/api/response/CategoryResponse.kt
package com.example.mhamed.demo.api.response

import com.example.mhamed.demo.model.Category

/**
 * Category object as represented by the server.
 * @author mhamed
 * @since 2018-11-10
 */
data class CategoryResponse(var _id: String, var updated_at:String, var custom_module_eid:String, var eid:String,
                            var __v:String, var created_at:String, var _active: Boolean,
                            var slug: String, override var title: String, override var description: String?) : Category {
}

File: QuickMobileTest/QuickMobileTest/Data/Category List/Entity/CategoryResponse.swift
//
//  CategoryResponse.swift
//  QuickMobileTest
//
//  Created by Balaji Galave on 5/18/19.
//

import Foundation

struct CategoryResponse: Codable {
    let _id : String = ""
    let updated_at : String?
    let slug : String?
    let custom_module_eid : String?
    let eid : String?
    let title : String?
    let description : String = ""
    let __v : Int = 0
    let _active : Bool = false
    let created_at : String?
}
